File: year2020/month09/day0909/ms_0909.cpp
#include <iostream>

using namespace std;

void MultTable(int* range);

int main() {
	int data[2];

	for (int i = 0; i < 2; i++)
		scanf_s("%d", &data[i]);

	if (data[0] > data[1]) {
		int temp = data[0];
		data[0] = data[1];
		data[1] = temp;
	}

	MultTable(data);

	return 0;
}

void MultTable(int* range) {
	for (int i = *range; i <= *(range + 1); i++) {
		cout << "== " << i << "dan ==" << endl;

		for (int j = 1; j <= 9; j++)
			printf("%d * %d = %2d\n", i, j, i * j);

		cout << endl;
	}
}

File: year2020/month09/day0909/ms_0909.cs
using System;

namespace algorithm_53
{
    class ms_0909
    {
        static void MultTable(int start, int end) 
        {
            for(int i = start; i <= end; i++) 
            {
                Console.WriteLine($"== {i}dan ==");
        
                for(int j = 1; j <= 9; j++)
                    Console.WriteLine($"{i} * {j} = {i*j, 2}");

                Console.WriteLine();
            }
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int start = Convert.ToInt32(input[0]);
            int end = Convert.ToInt32(input[1]);

            if(start > end) 
            {
                int temp = start;
                start = end;
                end = temp;
            }

            MultTable(start, end);
        }
    }
}
